Docker-Swarm-deployment/single-node/keycloak/keycloak-stack.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.4"
+
+networks:
+  default:
+    external:
+      name: overlay-net
+    driver: overlay
+    
+services:
+  postgresql:
+    image: "docker.io/bitnami/postgresql:11-debian-10"
+    environment:
+      - ALLOW_EMPTY_PASSWORD=yes
+      - POSTGRESQL_USERNAME=bn_keycloak
+      - POSTGRESQL_DATABASE=bitnami_keycloak
+    volumes:
+      - "postgresql_data:/bitnami/postgresql"
+  keycloak:
+    image: docker.io/bitnami/keycloak:12-debian-10
+    ports:
+      - "80:8080"
+    environment:
+      - KEYCLOAK_CREATE_ADMIN_USER=true
+    depends_on:
```

Comment:
depends on does not work in stack file/ swarm deployment. 

How is the keycloak connecting to postgres ? And also reuse postgres  stack filee defined in  single-node/postgres folder